components/popover_list_members/popover_list_members_item/popover_list_members_item.tsx

```diff
@@ -0,0 +1,106 @@
+import {UserProfile} from 'mattermost-redux/types/users';
+import {isGuest} from 'mattermost-redux/utils/user_utils';
+
+import ProfilePicture from 'components/profile_picture';
+import MessageIcon from 'components/widgets/icons/message_icon';
+import BotBadge from 'components/widgets/badges/bot_badge';
+import GuestBadge from 'components/widgets/badges/guest_badge';
+import SharedUserIndicator from 'components/shared_user_indicator';
+import CustomStatusEmoji from 'components/custom_status/custom_status_emoji';
+
+import {UserStatuses} from 'utils/constants';
+
+export type Props = {
+    showMessageIcon: boolean;
+    onItemClick: (user: UserProfile) => void;
+    status: string;
+    user: UserProfile;
+    displayName?: string;
+}
+
+PopoverListMembersItem.defaultProps = {
+    status: UserStatuses.OFFLINE,
+};
+
+export default function PopoverListMembersItem(props: Props) {
+    const handleClick = () => {
+        props.onItemClick(props.user);
+    };
+
+    if (!props.user) {
+        return null;
+    }
+
+    if (!props.displayName) {
+        return null;
+    }
+
+    let messageIcon;
+    if (props.showMessageIcon) {
+        messageIcon = (
+            <MessageIcon aria-hidden='true'/>
+        );
+    }
+
+    let sharedIcon;
+    if (props.user.remote_id) {
+        sharedIcon = (
+            <SharedUserIndicator
+                className='shared-user-icon'
+                withTooltip={true}
+            />
+        );
+    }
+    const botClass = props.user.is_bot ? ' more-modal__row--bot' : '';
+    const status = props.user.is_bot ? undefined : props.status;
+
+    return (
+        <div
+            data-testid='popoverListMembersItem'
+            tabIndex={0}
+            aria-label={props.displayName.toLowerCase()}
+            className={'more-modal__row' + botClass}
+            onClick={handleClick}
+        >
+            <ProfilePicture
+                src={Client4.getProfilePictureUrl(props.user.id, props.user.last_picture_update)}
+                status={status}
+                size='md'
+            />
+            <div className='more-modal__details d-flex whitespace--nowrap'>
+                <div className='more-modal__name'>
+                    <span>
+                        {props.displayName}
+                    </span>
+                    <BotBadge
+                        show={Boolean(props.user.is_bot)}
+                        className='badge-popoverlist'
+                    />
+                    <GuestBadge
+                        show={isGuest(props.user.roles)}
+                        className='badge-popoverlist'
+                    />
+                </div>
+                <CustomStatusEmoji
+                    userID={props.user.id}
+                    showTooltip={true}
+                    emojiSize={15}
+                    emojiStyle={{
+                        marginBottom: -3,
+                        marginLeft: 4,
+                    }}
```

Comment:
could we please add this as a `const` above, so it does not create a new object on every render?

Reply:
Done